benchmarks/bm/_to_bool.py

```diff
@@ -0,0 +1,38 @@
+def to_bool(val):
```

Comment:
We have a similar utility that we could probably use: https://github.com/DataDog/dd-trace-py/blob/5190cb4822c3410fd4d02f59097ebbed5e33986e/ddtrace/internal/utils/formats.py#L86

It's not quite as flexible but not sure that it needs to be _this_ flexible

Reply:
👋 Hi Kyle!
Thank you for your feedback!

[_to_bool.py can be removed after release of attrs 21.3.0](https://github.com/python-attrs/attrs/blob/e84b57ea687942e5344badfe41a2728e24037431/src/attr/converters.py#L114). This is a designed way to parse env vars in attrs (since August), but unfortunately not yet released; will be released in the next minor version of a library.

Given above, I don't mind updating to `asbool` though, if other folks think too this is a worthy change! 😃

Reply:
Also, referring to asbool will require an update in https://github.com/DataDog/dd-trace-py/blob/master/benchmarks/Dockerfile, because benchmarks don't have access to ddtrace internal code.